qa/Lisovik/1.1_Send_CV.feature

```diff
@@ -0,0 +1,168 @@
+	| valid   | invalid | filled      | attached     | not checked | error    |
+	| valid   | invalid | not filled  | not attached | not checked | error    |
+	| invalid | valid   | not filled  | not attached | not checked | error    |
+	| invalid | invalid | not filled  | attached     | not checked | error    |
+	| invalid | invalid | not filled  | not attached | checked     | error    |
+
+
+Scenario Outline: 1.1.2 Different datatypes in CV
+	Given other fields of CV form are filled with valid data
+	When user attaches <CV>
+	And user clicks Send button
+	Then <Result> <noerror_error> is expected
+	Examples:
+	|  CV  | Result  |
+	| docx | noerror |
+	| doc  | noerror |
+	| PDF  | noerror |
+	| RTF  | noerror |
+	| txt  | error   |
+	| exe  | error   |
+	| zip  | error   |
+	| rar  | error   |
+	| png  | error   |
+	| jpeg | error   |
+	|aaaaaa| error   |
+
+
+Scenario Outline: 1.1.3 Sizes of <CV> file
+	Given other fields of CV form are filled with valid data
+	When user attaches <CV>
+	And user clicks Send button
+	Then <Result> <noerror_error> is expected
+	Examples:
+	| CV      | Result  |
+	| 5 Mb    | noerror |
+	| 9 Mb    |  noerror|
+	| 9.9 Mb  | noerror |
+	| 10 Mb   | noerror |
+	| 10.1 Mb |  error  |
+	| 11 Mb   |  error  |
+	| 100 Mb  |  error  |
+
+
+
+Scenario: 1.1.4 Send an empty CV form
+	Given all fields of CV form are empty 
+	When user clicks Send button
+	Then Attach CV button and "I agree to the personal data processing" paint in red 
+	And red "This field is required" message appears
+
+Scenario Outline: 1.1.5 Lenght of message is less or more or equals 500 characters
+	When user enters <message> of <length> 
+	Then red <error_message> "The maximum number of characters is 500" <appears_noappears>
+	Examples:
+	| message | error_message |
+	| 0       | not appears   |
+	| 1       | not appears   |
+	| 100     | not appears   |
+	| 490     | not appears   |
+	| 499     | not appears   |
+	| 500     | not appears   |
+	| 501     | appears       |
+	| 510     | appears       |
+	| 900     | appears       |
+	| 1000    | appears       |
+
+Scenario Outline: 1.1.6 Format of Email is *@*.?*
+	When user enters <Email> of <format>
+	And numbers after "." are prohiibited
+	Then red <message> "Invalid email format" <appears_noappears>
+	Examples:
+	| Email         | message     |
+	| a@a.pa        | not appears |
+	| a@a.a         | appears     |
+	| aa.pa         | appears     |
+	| a             | appears     |
+	| a@a..pa       | appears     |
+	| a@@a.pa       | appears     |
+	| @a.pa         | appears     |
+	| 1@a.pa        | not appears |
+	| a@a.1a        | appears     |
+	| a@a.pa a@a.pa | appears     |
+	| a@a.pa 123    | appears     |
+
+
+Scenario Outline: 1.1.7 Entering different types of names in the Name field
+	When user enters <Name> of <format>
+	Then red <message> "Invalid name format" <appears_noappears>
+	Examples: 
+	| Name    |   message   |
+	|  A A    | not appears |
```

Comment:
Some values in Examples are duplicated and your covering is exceeded. 
You should use combine your data according to the selected test design technique. 
Also, you should separate scenarios for positive and negative scenarios.